rllib/algorithms/dqn/torch/dqn_rainbow_torch_learner.py

```diff
@@ -61,7 +61,7 @@ def compute_loss_for_module(
         ).squeeze()
 
         # Use double Q learning.
-        if self.config.double_q:
+        if config.double_q:
```

Comment:
Great catch. I had the same changed in another PR on the side. This would have been led to some bugs in MA off-policy.